bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -216,43 +216,24 @@ export class PlotCanvasView extends DOMView
     follow_enabled = false
     has_bounds = false
 
+    unioned_bounds =
+
+    r = null
     if @model.plot.match_aspect != false and @frame._width.value != 0 and @frame._height.value != 0
       r = 1/@model.plot.aspect_scale*(@frame._width.value/@frame._height.value)
```

Comment:
@mattpap Is there a better way to do this / communicate an optional ratio to the subsequent calls to `update` below?